android/src/main/java/com/razorpay/flutter_customui/RazorpayPlugin.java

```diff
@@ -107,28 +119,114 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
+      case "getBankAccount":
+        String bankStr = call.arguments.toString();
+        razorpayDelegate.getBankAccounts(bankStr,result,this.eventSink);
+        break;
+      case "getLinkedUpiAccounts":
+        razorpayDelegate.getLinkedUpiAccounts(call.arguments.toString(),result , this.eventSink);
+        break;
+      case "getBalance":
+        upiAccountStr = call.arguments.toString();
+        razorpayDelegate.getBalance( razorpayDelegate.getUpiAccount(upiAccountStr), result , this.eventSink);
+        break;
+      case "changeUpiPin":
+        upiAccountStr = call.arguments.toString();
+        razorpayDelegate.changeUpiPin( razorpayDelegate.getUpiAccount(upiAccountStr), result , this.eventSink);
+        break;
+      case "resetUpiPin":
+        _arguments = call.arguments();
+        upiAccountStr = (String) _arguments.get("upiAccount");
+        cardStr =  (String) _arguments.get("card");
+        razorpayDelegate.resetUpiPin( razorpayDelegate.getUpiAccount(upiAccountStr) , razorpayDelegate.getCard(cardStr) ,
+                result , this.eventSink);
+        break;
+      case "delink":
+        upiAccountStr = call.arguments.toString();
+        razorpayDelegate.delink( razorpayDelegate.getUpiAccount(upiAccountStr), result , this.eventSink);
+        break;
+      case "selectedBankAccount":
+        String bankAccountStr = call.arguments.toString();
+        razorpayDelegate.selectedBankAccount(razorpayDelegate.getBankAccount(bankAccountStr) ,result , this.eventSink);
+        break;
+      case "setUpUPIPin":
+         cardStr = call.arguments.toString();
+         razorpayDelegate.setupUpiPin(razorpayDelegate.getCard(cardStr) ,result , this.eventSink);
+         break;
+      case "isTurboPluginAvailable":
+         razorpayDelegate.isTurboPluginAvailable(result , this.eventSink);
+         break;
+        /*
+           Turbo TPV
+         */
+      case "linkNewUpiAccountTPV":
+        _arguments = call.arguments();
+        String customerId = (String) _arguments.get("customerId");
+        customerMobile = (String) _arguments.get("customerMobile");
+        orderId = (String) _arguments.get("orderId");
+        tpvBankAccount = (String) _arguments.get("tpvBankAccount");
+        razorpayDelegate.linkNewUpiAccount(customerMobile, customerId, orderId , tpvBankAccount , result , this.eventSink);
+        break;
+
+      case "linkNewUpiAccountWithUI":
+        _arguments = call.arguments();
+        customerMobile = (String) _arguments.get("customerMobile");
+        String color = (String) _arguments.get("color");
+        razorpayDelegate.linkNewUpiAccountWithUI(customerMobile, color , result , this.eventSink);
+        break;
+
+      case "manageUpiAccounts":
+        customerMobile = call.arguments.toString();
+        razorpayDelegate.manageUpiAccounts(customerMobile, result , this.eventSink);
+        break;
+
       default:
-        Log.d("RAZORPAY_SDK","no method");
+        Log.d(TAG,"no method");
```

Comment:
@AtifQamarRaz Please give a proper Log message here.

Reply:
@shibu000 it was already there. Should I change this?